python-avd/pyavd/_eos_cli_config_gen/j2templates/documentation/router-bgp.j2

```diff
@@ -51,9 +51,8 @@ ASN Notation: {{ router_bgp.as_notation | arista.avd.default('asplain') }}
 {%         endif %}
 {%         if router_bgp.bgp.additional_paths.receive is arista.avd.defined(true) %}
 | bgp additional-paths receive |
-{%         endif %}
-{%         if router_bgp.bgp.additional_paths.receive is arista.avd.defined(false) %}
-| no bgp additional-paths receive |
```

Comment:
I don't think this is necessary hence removed it.
@ClausHolbechArista

Reply:
I think we should leave it there:

```
router bgp 65000
+   no bgp additional-paths receive
    !
    address-family evpn
+      bgp additional-paths receive
```

please revert

Reply:
Done!